### ISSUE TITLE
Remove analytics from documentation

### ISSUE THREAD

**Issue by nawordar** [luafun/luafun#70]
Google Analytics is I think the most privacy invasive option that could have been chosen. I don't understand, why would analytics even be needed in documentation anyway.

**Comment by Totktonada** [comment]
I agree. I have no access to it and I doubt anybody even look into its results.
